builder/virtualbox/common/step_attach_guest_additions.go

```diff
@@ -77,7 +77,7 @@ func (s *StepAttachGuestAdditions) Cleanup(state multistep.StateBag) {
 		"--medium", "none",
 	}
 
-	if err := driver.VBoxManage(command...); err != nil {
-		ui.Error(fmt.Sprintf("Error unregistering guest additions: %s", err))
-	}
+	// Remove the ISO. Note that this will probably fail since
```

Comment:
oh, you mean the debug logs from the driver.vboxmanage? hmm, those aren't really supposed to be user facing, but I guess if you're trying to debug it could lead to some confusion. 

I think the complexity introduced by mutating state or checking for different flags is a more costly alternative than having some erroneous debug logs.